tests/cargo-kani/firecracker-block-example/README.md

```diff
@@ -0,0 +1,19 @@
+This example accompanies Kani's post on Firecracker. This describes a proof harness for ensuring that the Firecracker block device `parse` method adheres to a virtio requirement. We implement this as a standalone example with some simplifications (search for "Kani change" in the source).
```

Comment:
What is the requirement? It is mentioned several times, but I have not found a reference to it in here.

Reply:
Good call. I'll add the following about the requirement:

> 2.6.4.2 Driver Requirements: Message Framing
>
> The driver MUST place any device-writable descriptor elements after any device-readable descriptor elements.
>
> Source: https://docs.oasis-open.org/virtio/virtio/v1.1/csprd01/virtio-v1.1-csprd01.html#x1-280004